probe/docker/container.go

```
@@ -206,6 +207,7 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 	c.RLock()
 	defer c.RUnlock()
 
+	labels, _ := json.Marshal(c.container.Config.Labels)
```

Don't ignore this error.